blog/_posts/2022-01-19-learn-scala.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: blog-detail
+post-type: blog
+by: Julien Richard-Foy, Vincent Derouand, and Valérie Pedroni (Scala Center)
+title: "Learn Scala"
+---
+
+Happy new year 2022! The beginning of the year is a good time to refresh 
+your programming skills. We want to use this opportunity to report on what 
+happened on the MOOCs in 2021, and to share our plans for 2022.
+
+## 2021 Retrospective
+
+In 2021, with the release of Scala 3 we updated most of our MOOCs to Scala 3:
```

Comment:
nearly all of our MOOCs (instead of most of our MOOCs)

Reply:
Good catch! Addressed in 430b3bf0